service/device/registry/test-steps/src/main/java/org/eclipse/kapua/service/device/registry/steps/DeviceRegistrySteps.java

```diff
@@ -1084,6 +1084,7 @@ public void createConnections(List<CucConnection> cucConnections) throws Excepti
                 connectionCreator.setServerIp(cucConnection.getServerIp());
                 connectionCreator.setProtocol(cucConnection.getProtocol());
                 connectionCreator.setAllowUserChange(false);
+                connectionCreator.setAuthenticationType("USER_PASS");
 
```

Comment:
Not a big fun of this magic string, but I can understand it's difficult to do otherwise here